lib/redux/reducers/users.js

```diff
@@ -87,6 +88,18 @@ export function updateProfileErrorDismissed(state = null, action) {
   }
 }
 
+export function createCustodialAccountErrorMessage(state = null, action) {
+  switch (action.type) {
+    case actionTypes.CREATE_CUSTODIAL_ACCOUNT_FAILURE:
+      const err = action.payload;
+      return err.message;
+    case actionTypes.CREATE_CUSTODIAL_ACCOUNT_REQUEST:
```

Comment:
Just discussed this with Sara. She agrees the error should be dismissable ('x' in the corner) and the form validation errors should be surfaced with the same UI, although those should also auto-dismiss when the form validates as good. LMK if you have any questions tomorrow @krystophv...can explain more fully if needed.